apps/btshell/src/btshell.h

```diff
@@ -182,6 +185,36 @@ int btshell_l2cap_send(uint16_t conn, uint16_t idx, uint16_t bytes);
 int btshell_l2cap_reconfig(uint16_t conn_handle, uint16_t mtu,
                            uint8_t num, uint8_t idxs[]);
 
+#if (MYNEWT_VAL(BLE_ISO_BROADCASTER))
+int btshell_broadcast_base_add(uint8_t adv_instance, uint32_t presentation_delay);
```

Comment:
This is original concept to identify broadcast with adv_instance. Can one advertising instance be used for "multiple" purposes in context of one program ? Broadcast ID seems to be an unique ID

Reply:
Broadcast ID is indeed unique ID. Internally identifying broadcast with adv_instance (which is also used as BIG ID) saves memory. And AFAIK each adv instance can be only used for single broadcast either way - because it needs to have corelated Ext Adv and PA, and this cannot be started on already active instance. This means, you cannot start another broadcast on same instance.